tidb: fix config value for tiproxy (#15635)
pingcap/docs: tidb-configuration-file.md

### `session-token-signing-cert` <span class="version-mark">New in v6.4.0</span>

> + Cert file path, which is used by TiProxy for session migration.

Reviewer: ```suggestion
+ The certificate file path, which is used by TiProxy for session migration.
```

> + The certificate file path, which is used by TiProxy for session migration.

Reviewer: ```suggestion
+ The certificate file path, which is used by [TiProxy](/tiproxy-overview.md) for session migration.
```

> + The certificate file path, which is used by [TiProxy](/tiproxy-overview.md) for session migration.

Reviewer: ```suggestion
+ The certificate file path, which is used by [TiProxy](/tiproxy/tiproxy-overview.md) for session migration.
```

+ Default value: ""

> + Empty value will fail TiProxy session migration. To enable session migration, all TiDB nodes must set this to the same cert and key. That is, you should store the same cert and key on every TiDB node.

Reviewer: ```suggestion
+ Empty value will cause TiProxy session migration to fail. To enable session migration, all TiDB nodes must set this to the same cert certificate and key. This means that you should store the same certificate and key on every TiDB node.
```

> + Empty value will cause TiProxy session migration to fail. To enable session migration, all TiDB nodes must set this to the same cert certificate and key. This means that you should store the same certificate and key on every TiDB node.

Reviewer 2: ```suggestion
+ Empty value will cause TiProxy session migration to fail. To enable session migration, all TiDB nodes must set this to the same certificate and key. This means that you should store the same certificate and key on every TiDB node.
```

### `session-token-signing-key` <span class="version-mark">New in v6.4.0</span>

> + Key file path, which is used by TiProxy for session migration.

Reviewer: ```suggestion
+ The key file path, which is used by TiProxy for session migration.
```

+ Default value: ""

> + Refer descriptions of `session-token-signing-cert`.

Reviewer: ```suggestion
+ Refer to the descriptions of [`session-token-signing-cert`](#session-token-signing-cert-new-in-v640).
```

> + The key file path used by [TiProxy](/tiproxy-overview.md) for session migration.

Reviewer: ```suggestion
+ The key file path used by [TiProxy](/tiproxy/tiproxy-overview.md) for session migration.
```
